eng/scripts/tests/updateExistingOutput-expected.json

```diff
@@ -0,0 +1,25 @@
+{
+  "packages": [
+    {
+      "packageName": "Azure.IoT.DeviceUpdate",
+      "path": [
+        ".\\sdk\\deviceupdate\\Azure.IoT.DeviceUpdate"
+      ],
+      "language": ".Net",
+      "apiViewArtifact": ".\\artifacts\\packages\\Debug\\Azure.IoT.DeviceUpdate\\Azure.IoT.DeviceUpdate.1.0.0-alpha.20220726.1.nupkg",
+      "installInstructions": {
+        "full": "Download the Azure.IoT.DeviceUpdate package from [here](https://openapihub.test.azure-devex-tools.com/api/sdk-dl-pub?p=openapi-env-test/3146/azure-sdk-for-net-track2//Azure.IoT.DeviceUpdate.1.0.0-alpha.20220726.1.nupkg)",
+        "lite": "Download the Azure.IoT.DeviceUpdate package from [here](https://openapihub.test.azure-devex-tools.com/api/sdk-dl-pub?p=openapi-env-test/3146/azure-sdk-for-net-track2//Azure.IoT.DeviceUpdate.1.0.0-alpha.20220726.1.nupkg)"
+      },
+      "result": "succeeded",
+      "artifacts": [
+        ".\\artifacts\\packages\\Debug\\Azure.IoT.DeviceUpdate\\Azure.IoT.DeviceUpdate.1.0.0-alpha.20220726.1.nupkg"
+      ],
+      "changelog": {
+        "content": null,
+        "hasBreakingChange": false
+      },
+      "packageFolder": "C:/project/azure-sdk-for-net/sdk/deviceupdate/Azure.IoT.DeviceUpdate"
```

Comment:
This path looks specific to your computer and will likely fail if ran on another.

Reply:
the packageFolder is according to the running environment and the script will compute the path out. So all the outputJson files **-exptected.json may not exactly the same as the output of script. Those files are just for reference. I will update the files name to **-sample.json